src/views/ForgotPassword.vue

```diff
@@ -47,7 +49,13 @@ export default {
     async handleSubmit() {
       const valid = await this.$refs.form.validate()
 
-      if (valid) console.log("Form valid");
+      if (valid) {
+        try {
+          const result = await authService.forgotPassword(this.model);
```

Comment:
there is no need for storing this into a constant, just the call should be enough, if you want something extra you can add a loading state
async onSubmit() {
```
      try {
        this.loading = true
        await axios.post(`auth/users/reset_password`, this.model)
      } finally {
        this.loading = false
      }
    },
  },
```